### PR TITLE
Add Arp packet header

### PR DESCRIPTION
**Pull request by indraniBh**
Added support for ARP header

### CODE REVIEW

File: flow/packet-headers/arp.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+openapi: 3.1.0
+
+info:
+  title: Arp packet header model
+  version: ^0.0.0
+
+components:
+  schemas:
+    Flow.Arp:
+      description: Arp packet header
```
Comment (ankur-sheth) [inline flow/packet-headers/arp.yaml:10]: Change to ARP instead of Arp in the description.

**Review by ankur-sheth — CHANGES_REQUESTED** [review]

File: flow/packet-headers/arp.yaml (hunk 1 above)
Comment (ashutshkumr) [reply flow/packet-headers/arp.yaml:10]: This (all caps in description) will require change for other protocol descriptions as well. e.g. https://github.com/open-traffic-generator/models/blob/master/flow/packet-headers/udp.yaml#L10

(I agree with the change)

Comment (ankur-sheth) [reply flow/packet-headers/arp.yaml:10]: I didn't look closely at the other headers.  Glanced at a couple and they seemed ok.  Regardless, it should be ARP in the description.

Comment (indraniBh) [reply flow/packet-headers/arp.yaml]: Updated description.

**Review by ajbalogh — APPROVED** [review]

**Review by ankur-sheth — APPROVED** [review]

**Comment by ankur-sheth** [comment]
Merge ARP packet header model.

**Pull request merged**